utilities/blob_db/blob_db_impl.cc

```diff
@@ -1552,7 +1552,8 @@ Status BlobDBImpl::GetRawBlobFromFile(const Slice& key, uint64_t file_number,
 Status BlobDBImpl::Get(const ReadOptions& read_options,
                        ColumnFamilyHandle* column_family, const Slice& key,
                        PinnableSlice* value) {
-  return Get(read_options, column_family, key, value, nullptr /*expiration*/);
+  return Get(read_options, column_family, key, value,
+             (uint64_t*)nullptr /*expiration*/);
```

Comment:
Is `(uint64_t*)` necessary?

Reply:
yes, unfortunately it is needed to distinguish between DB::Get(const ReadOptions& options, ColumnFamilyHandle* column_family, const Slice& key, std::string* value, std::string* timestamp) and BlobDb::Get(const ReadOptions& options, const Slice& key, PinnableSlice* value, uint64_t* expiration).

Reply:
Will the following work?
```
return BlobDb::Get(read_options, column_family, key, value,
                                  nullptr /*expiration*/);
```

Reply:
It does not (at least with MSVC).

Reply:
Then let's use static_cast.